panel/models/quill.ts

```diff
@@ -57,7 +57,7 @@ export class QuillInputView extends HTMLBoxView {
 
   override render(): void {
     super.render()
-    this.container = div({style: "visibility: hidden;"})
+    this.container = div({style: {visibility: "hidden"}})
```

Comment:
I will restore `string` support in `style` attribute, because this is perfectly valid DOM.